tests/lib/mkpinentry.sh

```diff
@@ -1,6 +1,18 @@
 #!/bin/sh
 
-echo "setup fake gpg pinentry environment"
-mkdir -p ~/.snap/gnupg/
-echo pinentry-program "$TESTSLIB/pinentry-fake.sh" > ~/.snap/gnupg/gpg-agent.conf
-chmod -R go-rwx ~/.snap
+if [ gpg --version | MATCH "gpg (GnuPG) 1." ]; then
```

Comment:
It is true, I just saw the code, thanks for the heads up:
buf.WriteString(rc(true, "MATCH() { { set +xu; } 2> /dev/null; [ ${#@} -gt 0 ] || { echo \"error: missing regexp argument\"; return 1; }; local stdin=\"$(cat)\"; echo \"$stdin\" | grep -q -E \"$@\" || { echo \"error: pattern not found, got:\n$stdin\">&2; return 1; }; }\n"))